tests/unit/modules/test_file.py

```diff
@@ -981,7 +981,7 @@ def test_line_insert_before(self):
             '    - /srv/sugar'
         ])
         cfg_content = '- /srv/custom'
-        for before_line in ['/srv/salt', '/srv/sa.*t', '/sr.*']:
+        for before_line in ['/srv/salt', '/srv/sa.*t']:
```

Comment:
Why was the last regex removed here?

Reply:
Is moved to another test. Line 994. It's to general and should throw an exception.